dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/allgemein/rechtsstreitigkeiten/StatusZuEOptions.kt

```diff
@@ -0,0 +1,15 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.allgemein.rechtsstreitigkeiten
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * Enum class for the field statusZuE
+ */
```

Comment:
Do we need both Enums ("StatusZuEOptions and StatusZuGOptions)? Contentwise both are the same.

Reply:
sind auto generiert

Reply:
Klar könnte man überall ein und denselben enum benutzen.  Das devtool macht das hier jedoch für jede Frage automatisch neu, denn es weiß ja nicht, dass im Excel eine weitere row existiert, welche dieselben "Options" hat.
Ich würde das daher in dem Fall einfach hinnehmen.  Ist quasi ein Mini-Nachteil der Automatisierung des Ganzen.

Wenn du z.B. im Excel 50 Single-Select-Fragen hast, die alle das selbe set an Options haben (bspw. `["ja", "nein", "vielleicht", "manchmal"]`, dann würde das devtool diesen enum für jede Frage nochmal erstellen.
In so einem Extremfall könnte man nochmal überlegen, wie man damit umgeht.
Aber hier im gdv Framework finde ich es noch vertretbar, so wie es ist.

Reply:
Ja, ok. Verstehe jetzt woher das kommt. Finde ich hier ok, aber wenn man noch mehr ähnliche Enums produziert sollte man schauen ob man die nicht recyclen kann